### PR TITLE
📝 Added the heroku pointdns setup

### PR DESCRIPTION
**Pull request by LeTanque**
From notion

### CODE REVIEW

File: docs/topics/heroku-point-dns.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
+# Heroku PointDNS Setup
+
+Heroku doesn't offer nameservers. This means we must use a 3rd party
+DNS provider in order to assign a custom domain to an app. There are
+a number of ways to do this; the method described here is compatible
+with the Lambda Labs domain name process.
+
+## Objective
+
```
Comment (ryan-hamblin) [inline docs/topics/heroku-point-dns.md]: What's the objective? If you've listed a heading I'd recommend adding at least a bullet point here.
Something along the lines of:
- Establish a path to adding a custom domain for projects hosted on Heroku servers while remaining Labs Compliant.

Comment (tinomen) [reply docs/topics/heroku-point-dns.md]: Ah, I've got the template setup to break the objective up into 3 parts: summary, technologies and references. Are you suggesting that the summary be altered? I really like your wording better than what we came up with in the summary

Comment (tinomen) [reply docs/topics/heroku-point-dns.md]: Thinking we should get rid of the Summary section of Objective so the statement is directly after the objective header

**Comment by ryan-hamblin** [comment]
Looking good .I'll merge

**Comment by ryan-hamblin** [comment]
@LeTanque and @tinomen Just update the merge conflicts and I'll merge this bad-boy in.

**Pull request merged**